src/platform/nrfconnect/OTAImageProcessorImpl.cpp

```diff
@@ -67,6 +72,8 @@ CHIP_ERROR OTAImageProcessorImpl::Apply()
         err = dfu_target_schedule_update(-1);
     }
 
+    TriggerFlashAction(FlashHandler::Action::SLEEP);
```

Comment:
It's worth testing that change. I think Zigbee was fixing it, because if an app disable QSPI before rebooting, MCUboot may not be able to access it (unless our configuration fixes it in a different way).

Reply:
I did single DFU and application successfully booted after swap, but we can also run nightly on it to see if we are ok.

Reply:
OK, if you tested the image is swapped then we're good. Maybe our MCUboot configuration handles that somehow :)